cmd/blobstream-ops/verify/cmd.go

```diff
@@ -0,0 +1,216 @@
+package verify
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+
+	"github.com/celestiaorg/blobstream-ops/cmd/blobstream-ops/version"
+	"github.com/cosmos/cosmos-sdk/server"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	ethcmn "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/rs/zerolog"
+	"github.com/spf13/cobra"
+	blobstreamxwrapper "github.com/succinctlabs/blobstreamx/bindings"
+	tmconfig "github.com/tendermint/tendermint/config"
+	tmlog "github.com/tendermint/tendermint/libs/log"
+	"github.com/tendermint/tendermint/rpc/client/http"
+)
+
+// Command the verify command
+func Command() *cobra.Command {
+	orchCmd := &cobra.Command{
+		Use:          "verify",
```

Comment:
since there are a lot of things that could be verified, I'd suggest changing of this to be more specific, such as verfify-contract, but not blocking on it

Reply:
Good idea: [6c0faa8](https://github.com/celestiaorg/blobstream-ops/pull/3/commits/6c0faa885f879a1e1ee48550faa9aade96515ffb)